hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockStreamConfig.java

```diff
@@ -44,4 +49,31 @@ public record BlockStreamConfig(
         @ConfigProperty(defaultValue = "32") @NetworkProperty int hashCombineBatchSize,
         @ConfigProperty(defaultValue = "1") @NetworkProperty int roundsPerBlock,
         @ConfigProperty(defaultValue = "localhost") String grpcAddress,
-        @ConfigProperty(defaultValue = "8080") @Min(0) @Max(65535) int grpcPort) {}
+        @ConfigProperty(defaultValue = "8080") @Min(0) @Max(65535) int grpcPort,
+        @ConfigProperty(defaultValue = "3") @NetworkProperty int uploadRetryAttempts,
+        @ConfigProperty(defaultValue = "168") @NetworkProperty int localRetentionHours,
+        @ConfigProperty(defaultValue = "data/config/bucket-credentials.json") @NetworkProperty String credentialsPath,
+
+        // Bucket configurations with default AWS and GCP public buckets
+        @ConfigProperty(
+                        defaultValue =
+                                """
+        [
+            {
+                "name": "default-aws-bucket",
+                "provider": "AWS",
+                "endpoint": "https://s3.amazonaws.com",
+                "region": "us-east-1",
+                "bucketName": "hedera-mainnet-blocks"
```

Comment:
I think adding "enabled": "true" to each bucket would be useful to allow disabling/enabling each bucket individually at runtime.

Reply:
That's a good suggestion, thank you. Added!